src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -166,8 +188,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         getPossibleInterviewerIds(allAvailableInterviewers, interviewerCompany, interviewerJob);
     int randomNumber = (int) (Math.random() * possibleInterviewers.size());
     String interviewerId = possibleInterviewers.get(randomNumber);
-    // TODO: replace with real MeetLink from Calendar Access
-    String meetLink = "meet_link";
+
+    // Meet link is empty because we need a scheduledInterview before the link can be created
+    String meetLink = "";
```

Comment:
Why don't you just put this into the creation of the SchInt? Updating it after creation doesn't fit what the other fields are doing

Reply:
Done